From 04199db2aeea966c2c879b1764427c87433570f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Tue, 16 Aug 2016 10:37:28 +0300
Subject: wildmidi: Fix compilation with wildmidi 0.4

Old versions used char* for the data, new versions use int8_t*. They're often
not the same and differ in signedness.

diff --git a/ext/timidity/gstwildmidi.c b/ext/timidity/gstwildmidi.c
index 3e82ca2..f52ea9d 100644
--- a/ext/timidity/gstwildmidi.c
+++ b/ext/timidity/gstwildmidi.c
@@ -672,7 +672,7 @@ gst_wildmidi_get_buffer (GstWildmidi * wildmidi)
   gst_buffer_map (buffer, &info, GST_MAP_READWRITE);
 
   GST_OBJECT_LOCK (wildmidi);
-  size = WildMidi_GetOutput (wildmidi->song, (char *) info.data,
+  size = WildMidi_GetOutput (wildmidi->song, (gpointer) info.data,
       (unsigned long int) info.size);
   GST_OBJECT_UNLOCK (wildmidi);
 
-- 
cgit v0.10.2

